crates/matrix-sdk-crypto/src/olm/group_sessions/inbound.rs

```diff
@@ -106,6 +107,7 @@ impl InboundGroupSession {
         session_key: &SessionKey,
         encryption_algorithm: EventEncryptionAlgorithm,
         history_visibility: Option<HistoryVisibility>,
+        safety: KeySafety,
```

Comment:
Is this useful? You're always passing the same argument to the `new()` call, why not move it into the method?